pmd-kotlin/src/main/resources/category/kotlin/bestpractices.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+
+<ruleset name="Best Practices"
+    xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
+
+    <description>
+Rules which enforce generally accepted best practices.
+    </description>
+
+    <rule name="FunctionNameTooShort"
+          since="7.0"
+          language="kotlin"
+          message="Function names should have non-cryptic and clear names."
+          class="net.sourceforge.pmd.lang.rule.XPathRule"
+          typeResolution="true"
```

Comment:
typeresolution doesn't make sense here, since it is not there for kotlin

Reply:
ok, will remove it, I see default value is false.